### PR TITLE
Feat/condo/doma 4625/add isOutcome field to bank cost item

### PR DESCRIPTION
**Pull request by AntonAL**
Depends on #2870 

Please, look at the latest commit only

### CODE REVIEW

File: apps/condo/domains/banking/schema/BankTransaction.js [hunk 1]
```diff
@@ -75,19 +83,10 @@ const BankTransaction = new GQLListSchema('BankTransaction', {
         purpose: {
             schemaDoc: 'Textual description of payment purpose in free form',
             type: Text,
+            defaultValue: true,
```
Comment (sitozzz) [inline apps/condo/domains/banking/schema/BankTransaction.js:86]: Why default value of the field with `Text` type is become boolean?

File: apps/condo/domains/banking/schema/BankCostItem.test.js [hunk 2]
```diff
@@ -42,6 +42,8 @@ describe('BankCostItem', () => {
                 expect(obj.createdAt).toMatch(DATETIME_RE)
                 expect(obj.updatedAt).toMatch(DATETIME_RE)
                 expect(obj.name).toEqual(attrs.name)
+                expect(obj.isOutcome).toEqual(attrs.isOutcome)
+                expect(obj.outcome).toEqual(attrs.outcome)
```
Comment (sitozzz) [inline apps/condo/domains/banking/schema/BankCostItem.test.js:46]: There are no field with name outcome

File: apps/condo/domains/banking/schema/BankTransaction.js (hunk 1 above)
Comment (AntonAL) [reply apps/condo/domains/banking/schema/BankTransaction.js:86]: Oops, it is a bug

**Review by VKislov — APPROVED** [review]
You need to regenerate the migration. 219 is already in the master

**Review by sitozzz — CHANGES_REQUESTED** [review]
Please make single migration if it possible and rebase with origin branch to follow the migration number consistency

**Comment by VKislov** [comment]
> Please make single migration if it possible and rebase with origin branch to follow the migration number consistency

I think if you regenerate migrations, then one is created, if one of them is not manual of course

**Pull request closed without merging**